ecs.md

```diff
@@ -34,10 +34,19 @@ to set it up.
 
 To see the help message, execute the scripts with `-h` or `--help`.
 
-### Authentication and secrets
+### Quick start
+
+Follow the instructions in the `... > Instructions` menu from the Prodfiler UI, switching to the ECS tab.
+You will be guided through the execution of the scripts mentioned above.
+
+### Customize the setup
+
+If you need to adjust the default settings, follow along to know more about how the host-agent installation works.
```

Comment:
To put it simpler: (just a suggestion)
"The following describes how to change the default settings." ?

Reply:
Updated 🙏🏼